### ISSUE TITLE
contracts are not renamed when they have numbers in the name

### ISSUE THREAD

**Issue by nmushegian** [NexusDevelopment/dapple#25]
`dapple test` a contract named `Test1A`, for example

**Comment by ryepdx** [comment]
This appears to be working. I also don't see anything in the code that might be causing the bug you describe. Can you expand on what you did to trigger this bug and how you determined this is what was going wrong?

**Comment by ryepdx** [comment]
I've not been able to duplicate this bug.

**Comment by nmushegian** [comment]
I can't tell what's causing it, this is quite strange

```
Testing x8002f6579eeadbcd36a370309bc29021a35a7e847879648527b4754d76b09ee7_Test
  testAddBalance
  testAdminTransfer
  testSubBalance
  testSubBalanceFailsBelowZero
Testing MedianFinderTest
  testMoreFeedsThanSlots
    Used 136153 gas
  testOutOfOrderMedian
    Used 136303 gas
  testSimpleMedian
    Used 82448 gas
Testing x90fee5d07d332282d81b92eaf0f9c3d30b487c8cf94f33614bc6fef09129f3a4Test
  testOnlyOwnerCanTransfer
```

**Comment by nmushegian** [comment]
This is causing `dapple test --regex testname` to not match, which makes this a real issue

**Comment by ryepdx** [comment]
I believe this was resolved with commit df509e21d28f02e209e62d5057284c83c6e1ff48.
